vernac/declareUniv.mli

```diff
@@ -16,10 +16,15 @@ exception AlreadyDeclared of (string option * Id.t)
 
 (** Internally used to declare names of universes from monomorphic
    constants/inductives. Noop on polymorphic references. *)
-val declare_univ_binders : GlobRef.t -> UnivNames.universe_binders -> unit
+val declare_univ_binders : GlobRef.t -> UState.named_universes_entry -> unit
 
 (** Command [Universes]. *)
 val do_universe : poly:bool -> lident list -> unit
 
 (** Command [Constraint]. *)
 val do_constraint : poly:bool -> Constrexpr.univ_constraint_expr list -> unit
+
+(** Retrieve the set of monomorphic constraints generated by a global. This is
+    an underapproximation that should not be relied on for anything serious
+    but printing. *)
+val monomorphic_universes : GlobRef.t -> Univ.ContextSet.t option
```

Comment:
This API ends up unused so why have it?

Reply:
Leftover of a previous state, will remove it.